src/controllers/taskController.ts

```diff
@@ -0,0 +1,53 @@
+import { Request, Response } from 'express';
+import { createTask, deleteTask, getAllTask, getIdTask, updateTask } from '../services/taskService';
+
+
+const create = async (req: Request, res: Response) => {  
+    const newTask = await createTask(
```

Comment:
If you look in my PR, you can see how to do some really simple error handling. That'd probably be best to add before we merge this.